input/fsh/CancerRelatedMedicationStatement.fsh

```diff
@@ -8,7 +8,7 @@ It is important to note that cancer-related medications are not the ONLY medicat
 
 Conformance statement:
 
-MedicationStatement resources associated with an mCODE patient that are prescribed to treat the cancer condition SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
+MedicationStatement resources associated with an [mCODE patient](conformance.html#mCODE-Patients) SHOULD conform to this profile. Beyond this requirement, a producer of resources SHOULD ensure that any resource instance associated with an mCODE patient that would reasonably be expected to conform to this profile SHOULD be published in this form."
```

Comment:
I think the 2nd sentence can be removed -- it seems to me like it's redundant with the first sentence given we're saying all of the base resource SHOULD conform.

```suggestion
MedicationStatement resources associated with an [mCODE patient](conformance.html#mCODE-Patients) SHOULD conform to this profile."
```

Reply:
It is actually not redundant. 

The first statement covers MedicationStatement resources.

The second statement covers the case where a different resource is used for something that reasonably ought to have been a MedicationStatement.